web/src/utils/errorBoundry.tsx

```diff
@@ -0,0 +1,71 @@
+import {
+  Flex,
+  Stack,
+  Heading,
+  Button,
+  Text,
+  Link as ChakraLink,
+} from "@chakra-ui/react";
+import { Link } from "react-location";
+
+import React, { Component, ErrorInfo, ReactNode } from "react";
+import { ApprovalsLogo } from "../components/icons/Logos";
+
+interface Props {
+  children?: ReactNode;
+}
+
+interface State {
+  error: Error | undefined;
+}
+
+class ErrorBoundary extends Component<Props, State> {
+  public state: State = {
+    error: undefined,
+  };
+
+  public static getDerivedStateFromError(error: Error): State {
+    // Update state so the next render will show the fallback UI.
+    return { error: error };
+  }
+
+  public render() {
+    if (this.state.error != undefined) {
+      return (
+        <>
+          <Flex
+            height="100vh"
+            padding="0"
+            alignItems="center"
+            justifyContent="center"
+          >
+            <Stack textAlign="center" w="70%">
+              <Heading pb="10px">An unexpected error occurred.</Heading>
+              <Text>{this.state.error.message}</Text>
```

Comment:
```suggestion
             { this.state.error?.message &&  <Text>{this.state.error.message}</Text>}
```
Every error might not have an associated message with it. If the error object doesn't have message this will through a new error 😂

Reply:
Good catch @Eddie023 !